client/src/components/layouts/PageNotFound.css

```diff
@@ -0,0 +1,21 @@
+.error-layout-container {
+  margin-top: 2.8em;
+  height: calc(100vh - 2.8em);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+  background: linear-gradient(#1d1e27, rgb(78, 77, 78));
+}
+
+.error-layout__heading {
+  font-size: 12rem;
+  font-weight: bold;
+  margin: 0;
+}
+
+.error-layout__text {
+  font-size: 2rem;
+  text-align: center;
+  margin: 0;
+}
```

Comment:
- All files should have a copyright header.
- All files should have an EOF newline.
- Optionally, CSS rules should be sorted alphabetically.
- Prefer HEX to RGB.
- Class and ID names should be hyphen-delimited.

I'll see what I can do for automating EOF + CSS sorting/name checks.